remoting/README.md

```diff
@@ -0,0 +1,305 @@
+* POST-XML-MF: Input Arguments are mapped into XML-based files send in a multi-part request with content-type multipart/form-data (and XXX as boundary and application/json as Content-Disposition)
+
+Invocations can be batched. Normally ([Service Config](src/main/java/org/eclipse/tractusx/agents/remoting/config/ServiceConfig.java).batch is set to 1, no [Argument Config](src/main/java/org/eclipse/tractusx/agents/remoting/config/ArgumentConfig.java).formsBatchGroup is set to true) Remoting Agent will produce an outgoing REST call for each incoming tuple/binding. If [Service Config](src/main/java/org/eclipse/tractusx/agents/remoting/config/ServiceConfig.java).batch is greater than 1 or
+there is some [Argument Config](src/main/java/org/eclipse/tractusx/agents/remoting/config/ArgumentConfig.java).formsBatchGroup set to true, several tuples/bindings can be sent in a single invocation (usually in an array or by using flexible argument paths using '{<iriofinput>}' path elements). In that case, we also expect the responses to contain several individual results which are mapped/joined with the original input bindings using the ResultConfig.correlationInput reference.
+
+Invocation can be asynchronous. That means that the called backend will not return a proper response, just a successful notification code. Instead we send the public URL of the builtin [CallbackController](src/main/java/org/eclipse/tractusx/agents/remoting/callback/CallbackController.java) which is configured in the callbackAddress property of the remoting repository (and is transmitted in the callbackAddressProperty of the ServiceConfig). In order to correlate outgoing (batch) requests with asynchronous responses sent to the CallbackController, we rely on setting a unique request identifier specified in ServiceConfig.invocationIdProperty and comparing it with the content of the ResultConfig.callbackProperty
+ 
+## Deployment
+
+### Compile, Test & Package
+
+```console
+mvn package
+```
+
+This will generate 
+- a [standalone jar](target/remoting-agent-1.9.5-SNAPSHOT.jar) containing all necessary rdf4j components to build your own repository server.
+- a [pluging jar](target/original-remoting-agent-1.9.5-SNAPSHOT.jar) which maybe dropped into an rdf4j server for remoting support.
+
+### Run Locally
+
+The standalone jar](target/remoting-agent-1.9.5-SNAPSHOT.jar) contains an example application that runs a sample repository against a sample source
+
+```console
+java -jar target/remoting-agent-1.9.5-SNAPSHOT.jar -Dorg.slf4j.simpleLogger.defaultLogLevel=DEBUG
+```
+
+### Containerizing
+
+You could either call
+
+```console
+mvn install -Pwith-docker-image
+```
+
+or invoke the following docker command after a successful package run
+
+```console
+docker build -t ghcr.io/eclipse-tractusx/knowledge-agents/remoting-agent:1.9.5-SNAPSHOT -f src/main/docker/Dockerfile .
```

Comment:
Do not user ghcr.io but Docker Hub. This comment addresses all appearances of ghcr.io.

Reply:
switched to dockerhub/tractusx